at-quannguyen/W4/main.rb

```diff
@@ -0,0 +1,59 @@
+require './bbcnews.rb'
+require './cnnnews.rb'
+require './user.rb'
+
+bbc_news_01 = BbcNews.new(01, "BBC first news", "Hello world from BBC", "Say hello")
+bbc_news_02 = BbcNews.new(02, "BBC second news", "Hello Ruby from BBC", "Say hello")
+bbc_news_03 = BbcNews.new(03, "BBC third news", "Hello AsianTech from BBC", "Say hello")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/5cea93c6775631aa2d46f69d0bc35804ff430463/at-quannguyen/W4/main.rb#L7"></span>Naming/VariableNumber: Use normalcase for variable numbers.</td>
    </tr>
  </tbody>
</table>